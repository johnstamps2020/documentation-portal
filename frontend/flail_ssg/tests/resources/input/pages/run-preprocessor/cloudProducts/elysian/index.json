{
  "$schema": "/frontend/page-schema.json",
  "title": "Elysian",
  "template": "page",
  "class": "blue-theme elysian threeCards",
  "selector": {
    "label": "Select release",
    "selectedItem": "Elysian",
    "items": [
      {
        "label": "Dobson",
        "page": "../dobson"
      },
      {
        "label": "Cortina",
        "page": "../cortina"
      },
      {
        "label": "Banff",
        "page": "../banff"
      },
      {
        "label": "Aspen",
        "page": "../aspen"
      }
    ]
  },
  "items": [
    {
      "label": "Data and Analytics",
      "class": "categoryCard cardShadow",
      "items": [
        {
          "label": "Cloud Data Access",
          "page": "../cloudDataAccess/latest",
          "env": ["int", "staging"]
        },
        {
          "label": "DataHub",
          "page": "dhGwCloud/latest",
          "env": ["int", "staging"]
        },
        {
          "label": "InfoCenter",
          "page": "icGwCloud/latest",
          "env": ["int", "staging"]
        },
        {
          "label": "Guidewire Live",
          "class": "group",
          "items": [
            {
              "label": "Assess",
              "page": "assess"
            },
            {
              "label": "Canvas",
              "id": "canvas"
            },
            {
              "label": "Compare",
              "id": "comparelatest",
              "env": ["int", "staging"]
            },
            {
              "label": "Explore",
              "page": "../explore/latest",
              "env": ["int", "staging"]
            },
            {
              "label": "Predict",
              "id": "livepredictlatest",
              "env": ["int", "staging"]
            }
          ]
        }
      ]
    },
    {
      "label": "Feature Preview",
      "class": "categoryCard cardShadow",
      "env": ["int", "staging"],
      "items": [
        {
          "label": "Feature Preview",
          "page": "../featurePreview"
        }
      ]
    }
  ]
}
