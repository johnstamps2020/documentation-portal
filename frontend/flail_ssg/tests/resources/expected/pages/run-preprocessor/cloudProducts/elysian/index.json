{
  "$schema": "/frontend/page-schema.json",
  "title": "Elysian",
  "template": "page",
  "class": "blue-theme elysian threeCards",
  "selector": {
    "label": "Select release",
    "selectedItem": "Elysian",
    "items": [
      {
        "label": "Dobson",
        "page": "../dobson"
      }
    ]
  },
  "items": [
    {
      "label": "Data and Analytics",
      "class": "categoryCard cardShadow",
      "items": [
        {
          "label": "Guidewire Live",
          "class": "group",
          "items": [
            {
              "label": "Assess",
              "page": "assess"
            },
            {
              "label": "Canvas",
              "id": "canvas"
            }
          ]
        }
      ]
    }
  ]
}
